Création de l'annonce

gtedavid opened the pull request.

gtedavid committed:
ajoutc chgmt de nom de référent étudiant

gtedavid committed:
Correction Section Laura

noftaly commented on resources/global/2022-2023/Annonce de rentrée.md:
```diff
@@ -0,0 +1,68 @@
+
+**`Pour celles et ceux qui nous rejoignent`**
+Ef'Réussite est le serveur d'entraide édutiante pour les étudiants de l'Efrei.
+Géré par l'association Horizon, il permet ainsi à tout étudiant voulant donner de son temps pour aider ces camarades ou d'autres étudiants de sa promo qui auraient des difficultés, des questions en échange d'une reconnaissance pour sa participation dans ces objectifs d'entraide... 
```
```suggestion
Géré par l'association Horizon, il permet ainsi à tout étudiant voulant donner de son temps pour aider ses camarades ou d'autres étudiants de sa promo qui auraient des difficultés ou des questions, en échange d'une reconnaissance pour sa participation dans cet objectif d'entraide... 
```

noftaly commented on resources/global/2022-2023/Annonce de rentrée.md:
```diff
@@ -0,0 +1,68 @@
+
+*Liste les prof et leur préférence de contact ? à demander la permission auprès des enseignants*
+
+Fin juin dernier, le Pôle est ouvert à la possibilité de réaliser des cours en présentiel (5, 10 à 20 étudiants), et aussi à pouvoir organiser et permettre aux étudiants du Parcours Experts du Numérique à valoriser l'entraide réalisée auprès d'autres étudiants
```
```suggestion
Depuis juin, le Pôle est ouvert à la possibilité de réaliser des cours en présentiel (5, 10 à 20 étudiants), et à favoriser l'entraide aux étudiants du Parcours Experts du Numérique.
```

gtedavid replied:
Je crois que je l'ai corrigé je la mets comme résolu

noftaly commented on resources/global/2022-2023/Annonce de rentrée.md:
```diff
@@ -0,0 +1,68 @@
+
+On espère que vous avez passé un bel été et une bonne rentrée pour celles et ceux qui ont déjà repris 
+
+**`Pour celles et ceux qui nous rejoignent`**
```
```suggestion
:small_blue_diamond: **`Pour celles et ceux qui nous rejoignent`**

```

noftaly commented on resources/global/2022-2023/Annonce de rentrée.md:
```diff
@@ -0,0 +1,68 @@
+Les cours sur le Discord peuvent être enregistrés par les eProf. Les enregistrements sont stockés sur le [SharePoint d'Horizon](https://bit.ly/EfRéussiteSP) * 
+Certains cours peuvent être réalisés sur Teams si l'eProf assurant le cours rencontre des soucis techniques.
+
+Les Responsables eProf aide le responsable de Pôle dans la gestion du serveur, la planification et l'organisation des cours.
+
+Le Pôle Ef'Réussite s'est restructuré depuis l'année dernière selon se schéma :
+  - <@890084641317478400>, M2 Network & Cloud Infrastructure, Responsable de Pôle
+  - <@362902765124452364>, L3 classique, Co-Responsable de Pôle
+  - <@542818109912907776>, L3 Int, Responsable eProf L3
+  - <@602744407417946123>, L2 Int, Responsable eProf L2
+  - Un responsable eProf L1, en L1 classique/int/r : MP <@890084641317478400> ou [remplis le form](https://bit.ly/EfRéussiteRecrute) si ça vous intéresse
+  - Un co-responsable eProf L1, en L2 pour pouvoir seconder et accompagner le respo eprof L1 à déterminer les matières qui auraient besoin de cours : MP <@890084641317478400> ou [remplis le form](https://bit.ly/EfRéussiteRecrute) si ça vous intéresse
+  - <@226846557222207488>, M1 Cybersécurité, infrastructure et logiciel, Responsable SharePoint
+
+Tu pourras aussi trouver des membres de l'asso dans le Discord, ils ont le rôle <@&805883190878928907>
```
Je ne suis pas convaincu de la pertinence, ou j'en parlerai plus brièvement. IMO c'est trop détaillé, et on peut juste renvoyer vers le message dans #guide-et-liens pour ceux qui souhaitent connaître l'organisation interne du pôle

gtedavid replied:
mmm
ok
mais on le mettrait à la fin du canal ?

gtedavid replied:
!!! comment outdated, j'ai modif y a peu

SaadaLaura replied:
Pour moi ce serait bien de le mettre dans le message (à la fin ou au milieu) car ça peut aider les étudiants (nouveaux ou non) à savoir qui contacter. 
Perso j'entends beaucoup de gens dire qu'ils ne savent pas qui contacter au sein de l'admin en cas de pb donc la au moins ils savent qui sont les personnes qui ont un rôle de respo ...

gtedavid replied:
le souci, c'est que je ne sais pas si Elliot a modifié qlq ch dans sa proposition ou a juste ajouté le texte ? 😅 
Parce que c'est une ancienne version qui est commentée
@noftaly si tu as apporté des suggestions de modifications tu peux re annoter si tu veux

SaadaLaura replied:
J'ai pas l'impression je pense qu'il voulait juste te montrer le passage avec lequel il n'était pas d'accord

gtedavid replied:
le positionnement est bon dans l'annonce d'après toi @lauragossip ?

> Pour moi ce serait bien de le mettre dans le message (à la fin ou au milieu) car ça peut aider les étudiants (nouveaux ou non) à savoir qui contacter. Perso j'entends beaucoup de gens dire qu'ils ne savent pas qui contacter au sein de l'admin en cas de pb donc la au moins ils savent qui sont les personnes qui ont un rôle de respo ...

Est-ce qu'on devrait séparer les pings pour les promos dans les canaux de promo ?

SaadaLaura replied:
> le positionnement est bon dans l'annonce d'après toi @lauragossip ?
> 
> > Pour moi ce serait bien de le mettre dans le message (à la fin ou au milieu) car ça peut aider les étudiants (nouveaux ou non) à savoir qui contacter. Perso j'entends beaucoup de gens dire qu'ils ne savent pas qui contacter au sein de l'admin en cas de pb donc la au moins ils savent qui sont les personnes qui ont un rôle de respo ...
> 
> Est-ce qu'on devrait séparer les pings pour les promos dans les canaux de promo ?
Perso je pensais que tu voulais faire ça pour les pings, c'est a dire chaque canal à son ping ...

Pour la position il est a la fin (si on ne compte pas les pings des promo) donc oui il est bien placé

gtedavid replied:
Ok je résous cette conversation

gtedavid replied:
On verra ce que ça donne en draft

noftaly commented on resources/global/2022-2023/Annonce de rentrée.md:
```diff
@@ -0,0 +1,68 @@
+
+Tu pourras aussi trouver des membres de l'asso dans le Discord, ils ont le rôle <@&805883190878928907>
+
+**`Pour tout le monde`**
```
```suggestion
:small_blue_diamond: **`Pour tout le monde`**
```

noftaly commented on resources/global/2022-2023/Annonce de rentrée.md:
```diff
@@ -0,0 +1,68 @@
+Ci-dessous quelques infos à savoir par rapport à Ef'Réussite et l'Efrei pour bien commencer l'année
+ - les messages dans les canaux <#817741515564122143> et <#892407797961089114> ont été mis à jour
+   Ping <@&701774653940564078> si quelque chose n'est pas clair
+ - les rôles pour accéder au serveur restent les mêmes Efrei -> Pacours Ingé / Parcours Pex -> Rôles Ingé ou Rôles Pex + un canal de rôles supplémentaires
```
```suggestion
 - les rôles pour accéder au serveur restent les mêmes : Efrei → Pacours Ingé/Pex → Rôles Ingé/Pex, ainsi qu'un canal de rôles supplémentaires
```

noftaly commented on resources/global/2022-2023/Annonce de rentrée.md:
```diff
@@ -0,0 +1,68 @@
+Bienvenue en L2 !
+Allez plus qu'un an avant l'expérience à l'étranger, ça va être une expérience à se raconter des histoires même 10 ans après !
+On charbonne encore
+En Physique l'année dernière, votre programme a changé, Ef'Réussite a eu du mal à répondre présent 😔, alors c'est **TA chance** pour aider à ce que ça chaneg cette année !
```
```suggestion
Le programme de physique a changé l'année dernière et Ef'Réussite a eu du mal à répondre présent 😔, alors c'est **TA chance** pour aider à ce que ça change cette année !
```

noftaly commented on resources/global/2022-2023/Annonce de rentrée.md:
```diff
@@ -0,0 +1,68 @@
+
+ <@&799636419249045554> <@&810540280155471913>
+Bienvenue en M1, les anciens L3
+Alors ça commence à se professionnaliser ? Trouver un stage c'est difficile 😩 Postuler et Postuler disaient-ils...
```
```suggestion
Alors ça commence à se professionnaliser ? Trouver un stage c'est difficile 😩 Postuler et postuler disaient-ils...
```

noftaly commented on resources/global/2022-2023/Annonce de rentrée.md:
```diff
@@ -0,0 +1,68 @@
+ <@&799636419249045554> <@&810540280155471913>
+Bienvenue en M1, les anciens L3
+Alors ça commence à se professionnaliser ? Trouver un stage c'est difficile 😩 Postuler et Postuler disaient-ils...
+Allez on croit en vous, vous avez d'évoluer et c'est ça qu'ils cherchent : que vous êtes intéressés par un projet dans l'entreprise, dans l'entreprise même....
```
"vous avez d'évoluer" ? je ne suis pas sûr de voir ce que tu as voulu dire 😅 
```suggestion
Allez on croit en vous, vous avez envie d'évoluer et c'est ça qu'ils cherchent : montrez que vous êtes intéressés par un projet dans l'entreprise ou par l'entreprise même...
```

gtedavid replied:
envie oups

gtedavid replied:
tu en penses quoi une fois ajouter ? 😅
j'ai peut-être pas les bons "termes" ou ai du mal à trouver les "bons mots" pour qualifier ce que je veux dire....

Motivé ?

gtedavid replied:
@lauragossip tu en penses quoi de ces phrases ?

SaadaLaura replied:
C'est pas mal je sais pas si c'est utile de rajouter un "En tout cas vous allez peut-être changer mais nous pas, on sera toujours là pour vous aider si vous avez du mal à trouver un stage ou dans une matière..." 
Mais je ne sais pas si ça fait pas un peut lourd

gtedavid replied:
> C'est pas mal je sais pas si c'est utile de rajouter un "En tout cas vous allez peut-être changer mais nous pas, on sera toujours là pour vous aider si vous avez du mal à trouver un stage ou dans une matière..." Mais je ne sais pas si ça fait pas un peut lourd

aider pour un stage pour le moment, ça risque d'être difficile 😅

gtedavid replied:
J'essayais d'avoir un msg d'encouragement, positif

gtedavid replied:
J'ai rajouté "envie"
mais on peut probablement améliorer la phrase

noftaly reviewed (changes requested):
voilà my 2 cents

Je serai pour mettre les annonces promo-specific dans le salon d'annonce de la promo, ca fera une annonce globale moins grosse et donc beaucoup plus lisible 😅

gtedavid committed:
MAJ info + et logo changement

gtedavid committed:
Corrections L7

Co-authored-by: Elliot <elliot.maisl@gmail.com>

gtedavid committed:
Diamond L33

Co-authored-by: Elliot <elliot.maisl@gmail.com>

gtedavid committed:
Corrections + Ajustements flèches

Co-authored-by: Elliot <elliot.maisl@gmail.com>

gtedavid committed:
Améliorations du texte des L2

Co-authored-by: Elliot <elliot.maisl@gmail.com>

gtedavid committed:
Diamond L5

Co-authored-by: Elliot <elliot.maisl@gmail.com>

gtedavid commented:
Lien du form changera dès que les forms seront mis sur Okampus (Ivan me tiendra au courant à ce niveau-là 😅 
@lauragossip @noftaly @melisandeGB

gtedavid committed:
Correct°.

Co-authored-by: Elliot <elliot.maisl@gmail.com>

gtedavid committed:
Diamond L21

gtedavid committed:
L14 ajustements

gtedavid committed:
A la ligne pr L22

SaadaLaura commented on resources/global/2022-2023/Annonce de rentrée.md:
```diff
@@ -0,0 +1,80 @@
+
+:small_blue_diamond: **`Pour celles et ceux qui nous rejoignent`**
+
+Ef'Réussite est le serveur d'entraide édutiante pour les étudiants de l'Efrei.
```
Tu fera attention tu as écrit "édutiante" au lieu de "étudiante" 
```suggestion
Ef'Réussite est le serveur d'entraide étudiante pour les étudiants de l'Efrei.
```

SaadaLaura commented on resources/global/2022-2023/Annonce de rentrée.md:
```diff
@@ -0,0 +1,80 @@
+Les cours sur le Discord peuvent être enregistrés par les eProf. Les enregistrements sont stockés sur le [SharePoint d'Horizon](https://bit.ly/EfRéussiteSP) * 
+Certains cours peuvent être réalisés sur Teams si l'eProf assurant le cours rencontre des soucis techniques.
+
+Les Responsables eProf aide le responsable de Pôle dans la gestion du serveur, la planification et l'organisation des cours.
+
+:small_blue_diamond:**`Composition du Pôle`**
+
+Le Pôle Ef'Réussite s'est restructuré depuis l'année dernière selon se schéma :
```
J'ai vu deux petites fautes mais je ne suis pas sûr pour le "ce/se" : 
```suggestion
Les Responsables eProf aident le responsable de Pôle dans la gestion du serveur, la planification et l'organisation des cours.

:small_blue_diamond:**`Composition du Pôle`**

Le Pôle Ef'Réussite s'est restructuré depuis l'année dernière selon ce schéma :
```

gtedavid replied:
Nope tu as raison
good catch

SaadaLaura commented on resources/global/2022-2023/Annonce de rentrée.md:
```diff
@@ -0,0 +1,80 @@
+ - les messages dans les canaux <#817741515564122143> et <#892407797961089114> ont été mis à jour
+   Ping <@&701774653940564078> si quelque chose n'est pas clair
+ - les rôles pour accéder au serveur restent les mêmes : Efrei → Pacours Ingé/Pex → Rôles Ingé/Pex, ainsi qu'un canal de rôles supplémentaires
+ - On est jamais fermé ! On est ouvert à toutes et à tous, curieux, soucieux d'aider sans savoir comment. On finit toujours par trouver un nouveau moyen innovant
+ - le responsable réussite étudiante devient le responsable de formation
+
+Et qui dit rentrée.... Dit nouveautés !
+On se met au couleur de l'association, on sait certains aimer l'ancien logo, on le garde dans notre cœur et on dit bienvenue au nouveau logo !
```
Pour la négation je sais pas si tu veux mettre le "ne" mais dans le doute je te le met en commentaire...
```suggestion
 - On n'est jamais fermé ! On est ouvert à toutes et à tous, curieux, soucieux d'aider sans savoir comment. On finit toujours par trouver un nouveau moyen innovant
 - le responsable réussite étudiante devient le responsable de formation

Et qui dit rentrée.... Dit nouveautés !
On se met aux couleurs de l'association, on sait certains aimer l'ancien logo, on le garde dans notre cœur et on dit bienvenue au nouveau logo !
```

gtedavid replied:
RAS
bv

SaadaLaura commented on resources/global/2022-2023/Annonce de rentrée.md:
```diff
@@ -0,0 +1,80 @@
+ <@&998640012537122917> <@&998640714281910344>
+Bienvenue à l'Efrei 🥳 
+Sens toi libre de poser tes questions dans les canaux appropriés (<#892407797961089114> pour te repérer)
+On
```
Est-ce que il y a une phrase à ajouter ?

gtedavid replied:
justement @lauragossip 
Je ne sais pas trop quoi ajouter 😅 
donc si tu as des propositions, des idées

gtedavid replied:
le samedi 8 septembre ?

SaadaLaura replied:
en vrai tu pourrais en parler mais je sais pas si ivan serait d'accord ...

J'avoue que je sèche aussi les seules idées que j'ai rejoignent un peut la phrase juste au dessus

gtedavid replied:
j'ai rajouté une phrase @lauragossip @melisandeGB 
voir la dernière version

SaadaLaura commented on resources/global/2022-2023/Annonce de rentrée.md:
```diff
@@ -0,0 +1,80 @@
+
+
+ <@&794628669737009183> <@&810540179609092096>
+Bienvenue en L... Ah pardon, vous allez à l'étranger 😅 Alors éclatez-vous, mais pas trop, on aimerait vous voir entier mi-décembre.
```
Il manque un mot selon moi 
```suggestion
Bienvenue en L... Ah pardon, vous allez à l'étranger 😅 Alors éclatez-vous, mais pas trop, on aimerait vous voir en entier mi-décembre.
```

gtedavid replied:
bv 👍

gtedavid committed:
Correction typo

Co-authored-by: lauragossip <78168221+lauragossip@users.noreply.github.com>

gtedavid committed:
Corrections lignes 19 et 23

Co-authored-by: lauragossip <78168221+lauragossip@users.noreply.github.com>

gtedavid committed:
Corrections L41 & 45

Co-authored-by: lauragossip <78168221+lauragossip@users.noreply.github.com>

gtedavid committed:
Mot manquant L 62

Co-authored-by: lauragossip <78168221+lauragossip@users.noreply.github.com>

gtedavid committed:
Rajout L70 "en" et rajout L51

gtedavid committed:
Inversement L8 ? & difficultés

gtedavid committed:
Ajout des lignes 59 et 60

Dépose des copies, cours, exercices de TD dans les channels docs L1 pour les étudiant(e)s de la Promo 2027, mais aussi pour les années futures ! Tu peux les annonymiser évidemment ;)
Si tu rencontres des soucis ou n'est pas sûr comment pour accéder ou déposer les documents, contactes <@&934147334760169574>

gtedavid commented:
@noftaly @lauragossip @melisandeGB 
j'ai ajouté les lignes 59 et 60 
Dites-moi ce que vous en pensez ;)

gtedavid committed:
Style L38 à 45 - Ajout ligne 43 Bot

iv-stpn commented on resources/global/2022-2023/Annonce de rentrée.md:
```diff
@@ -0,0 +1,84 @@
+Hello à toutes et à tous 👋 :)
```
```suggestion
Hello à toutes et à tous 👋 !
```

iv-stpn commented on resources/global/2022-2023/Annonce de rentrée.md:
```diff
@@ -0,0 +1,84 @@
+Hello à toutes et à tous 👋 :)
+
+On espère que vous avez passé un bel été et une bonne rentrée pour celles et ceux qui ont déjà repris 
```
```suggestion
On espère que vous avez passé un bel été et nous vous souhaitons une bonne rentrée !
```

iv-stpn commented on resources/global/2022-2023/Annonce de rentrée.md:
```diff
@@ -0,0 +1,84 @@
+
+On espère que vous avez passé un bel été et une bonne rentrée pour celles et ceux qui ont déjà repris 
+
+:small_blue_diamond: **`Pour celles et ceux qui nous rejoignent`**
```
```suggestion
:small_blue_diamond: **`Pour les nouveaux 🌱`**
```

iv-stpn commented on resources/global/2022-2023/Annonce de rentrée.md:
```diff
@@ -0,0 +1,84 @@
+
+:small_blue_diamond: **`Pour celles et ceux qui nous rejoignent`**
+
+Ef'Réussite est le serveur d'entraide étudiante pour les étudiants de l'Efrei.
```
```suggestion
Ef'Réussite est le serveur d'entraide étudiante de l'Efrei.
```

iv-stpn commented on resources/global/2022-2023/Annonce de rentrée.md:
```diff
@@ -0,0 +1,84 @@
+:small_blue_diamond: **`Pour celles et ceux qui nous rejoignent`**
+
+Ef'Réussite est le serveur d'entraide étudiante pour les étudiants de l'Efrei.
+Géré par l'association Horizon, il permet ainsi à tout étudiant voulant donner de son temps pour aider ses camarades ou d'autres étudiants de sa promo qui auraient des questions ou des difficultés, en échange d'une reconnaissance pour sa participation dans cet objectif d'entraide... 
```
```suggestion
Il est géré par l'association <@&805883190878928907>, par des étudiants bénévoles souhaitant aider leurs pairs (appelés <@&805873100813500447> ou eProfs). 

Nous proposons :
- des séances de révisions en ligne sur Ef'Réussite 💻
- des séances en présentiel sous la forme de cours ou de [mentorat en groupe](https://discord.gg/G7fWxQZXqF) 📚
- des enregistrements de ces séances stockés sur le [SharePoint d'Horizon](https://bit.ly/EfRéussiteSP) 🔗
- des ressources pédagogiques disponibles dans les canaux "-docs" 📄
- du [soutien pour vos projets de développement](https://discord.gg/TZ7VUHE9yf) et pour aller plus loin en programmation ! 🚀
```

gtedavid replied:
```suggestion
Il est géré par les l'association <@&805883190878928907>, par des étudiants bénévoles, appelés <@&805873100813500447> ou eProfs, souhaitant aider leurs pairs. 

Nous proposons :
- des séances de révisions en ligne sur Ef'Réussite 💻
- des séances en présentiel suivant la demande et le nombre d'étudiants intéressés
- des séances en présentiel de [mentorat en groupe assurées par le Pôle Mentorat](https://discord.gg/G7fWxQZXqF)
- des enregistrements de ces séances stockés sur le [SharePoint d'Horizon](https://bit.ly/EfRéussiteSP) 🔗
- des ressources pédagogiques disponibles dans les canaux "-docs" 📄
- du [soutien pour vos projets de développement](https://discord.gg/TZ7VUHE9yf) et pour aller plus loin en programmation ! 🚀
```
Suggestion bis but not taken into account - redoing whole announcement

iv-stpn commented on resources/global/2022-2023/Annonce de rentrée.md:
```diff
@@ -0,0 +1,84 @@
+
+Ef'Réussite est le serveur d'entraide étudiante pour les étudiants de l'Efrei.
+Géré par l'association Horizon, il permet ainsi à tout étudiant voulant donner de son temps pour aider ses camarades ou d'autres étudiants de sa promo qui auraient des questions ou des difficultés, en échange d'une reconnaissance pour sa participation dans cet objectif d'entraide... 
+Certains enseignants de l'Efrei sont dans le Discord. 
```
```suggestion
**NOTE: des enseignants et membres de l'administration de l'Efrei sont présents sur le Discord.**
```

iv-stpn commented on resources/global/2022-2023/Annonce de rentrée.md:
```diff
@@ -0,0 +1,84 @@
+Géré par l'association Horizon, il permet ainsi à tout étudiant voulant donner de son temps pour aider ses camarades ou d'autres étudiants de sa promo qui auraient des questions ou des difficultés, en échange d'une reconnaissance pour sa participation dans cet objectif d'entraide... 
+Certains enseignants de l'Efrei sont dans le Discord. 
+
+*Liste les prof et leur préférence de contact ? à demander la permission auprès des enseignants*
```
Yes, surtout CMB & Zalaméa
Les autres n'utilisent jamais Discord

gtedavid replied:
J'ai contacté CMB, mais il faut qu'il réponde à ma réponse, je veux juste bien qu'il comprenne le but de mettre cette info là

Pour le mail, je pense l'envoyer demain?

iv-stpn commented on resources/global/2022-2023/Annonce de rentrée.md:
```diff
@@ -0,0 +1,84 @@
+*Liste les prof et leur préférence de contact ? à demander la permission auprès des enseignants*
+
+
+Depuis juin, le Pôle est ouvert à la possibilité de réaliser des cours en présentiel (5, 10 à 20 étudiants), et à favoriser l'entraide aux étudiants du Parcours Experts du Numérique.
```
```suggestion
Horizon expérimente avec la réalisation de cours en présentiel, et avec un programme pour favoriser l'entraide pour les __étudiants <@&809188399248179201>__
```

gtedavid replied:
mmm
j'aimerais gardé la notion de date quand même.....

gtedavid replied:
```suggestion
Ef'Réussite expérimente depuis juin la réalisation de cours en présentiel pour favoriser l'entraide entre __étudiants__
```

gtedavid replied:
prq tu ping les pex dessus ???

noftaly replied:
Le but etait de dire "... favoriser l'entraide entre étudiants PEx", donc il a mentionné le rôle pour mettre de la couleur 😄

iv-stpn commented on resources/global/2022-2023/Annonce de rentrée.md:
```diff
@@ -0,0 +1,84 @@
+
+
+Depuis juin, le Pôle est ouvert à la possibilité de réaliser des cours en présentiel (5, 10 à 20 étudiants), et à favoriser l'entraide aux étudiants du Parcours Experts du Numérique.
+On appelle les étudiants donnant des cours, les <@805873100813500447> ou eProf ; celles et ceux étant ouverts à des cours en présentiel ont le rôle <@&821921661661610004>.
```
```suggestion
```

iv-stpn commented on resources/global/2022-2023/Annonce de rentrée.md:
```diff
@@ -0,0 +1,84 @@
+
+Depuis juin, le Pôle est ouvert à la possibilité de réaliser des cours en présentiel (5, 10 à 20 étudiants), et à favoriser l'entraide aux étudiants du Parcours Experts du Numérique.
+On appelle les étudiants donnant des cours, les <@805873100813500447> ou eProf ; celles et ceux étant ouverts à des cours en présentiel ont le rôle <@&821921661661610004>.
+Les cours sur le Discord peuvent être enregistrés par les eProf. Les enregistrements sont stockés sur le [SharePoint d'Horizon](https://bit.ly/EfRéussiteSP) * 
```
```suggestion
Les cours ont lieu sur Discord ou Teams et sont généralement enregistrés par les <@&805873100813500447>. Si vous souhaitez qu'une de vos interventions soit supprimée du rendu public de la vidéo, que votre nom n'y apparaisse pas, ou autre, envoyez une demande au <@&1012394085174227045>.
```

iv-stpn commented on resources/global/2022-2023/Annonce de rentrée.md:
```diff
@@ -0,0 +1,84 @@
+Depuis juin, le Pôle est ouvert à la possibilité de réaliser des cours en présentiel (5, 10 à 20 étudiants), et à favoriser l'entraide aux étudiants du Parcours Experts du Numérique.
+On appelle les étudiants donnant des cours, les <@805873100813500447> ou eProf ; celles et ceux étant ouverts à des cours en présentiel ont le rôle <@&821921661661610004>.
+Les cours sur le Discord peuvent être enregistrés par les eProf. Les enregistrements sont stockés sur le [SharePoint d'Horizon](https://bit.ly/EfRéussiteSP) * 
+Certains cours peuvent être réalisés sur Teams si l'eProf assurant le cours rencontre des soucis techniques.
```
```suggestion
```

iv-stpn commented on resources/global/2022-2023/Annonce de rentrée.md:
```diff
@@ -0,0 +1,84 @@
+On appelle les étudiants donnant des cours, les <@805873100813500447> ou eProf ; celles et ceux étant ouverts à des cours en présentiel ont le rôle <@&821921661661610004>.
+Les cours sur le Discord peuvent être enregistrés par les eProf. Les enregistrements sont stockés sur le [SharePoint d'Horizon](https://bit.ly/EfRéussiteSP) * 
+Certains cours peuvent être réalisés sur Teams si l'eProf assurant le cours rencontre des soucis techniques.
+
```
```suggestion
```

iv-stpn commented on resources/global/2022-2023/Annonce de rentrée.md:
```diff
@@ -0,0 +1,84 @@
+Les cours sur le Discord peuvent être enregistrés par les eProf. Les enregistrements sont stockés sur le [SharePoint d'Horizon](https://bit.ly/EfRéussiteSP) * 
+Certains cours peuvent être réalisés sur Teams si l'eProf assurant le cours rencontre des soucis techniques.
+
+Les Responsables eProf aident le responsable de Pôle dans la gestion du serveur, la planification et l'organisation des cours.
```
```suggestion
Les <@&938679049746792448> gèrent Ef'Réussite ; contactez-les pour toute question relative à la planification et l'organisation de cours, ou pour vous [engager](https://bit.ly/EfRéussiteRecrute) comme <@&805873100813500447> !
Les canaux ` 🟠-cours-LX` comportent les annonces des cours auxquels tu peux t'inscrire aux rappels 🔔 
```

gtedavid replied:
faute et en relecture

gtedavid replied:
J'ai rajouté une ligne sur ta suggestion @iv-stpn

iv-stpn commented on resources/global/2022-2023/Annonce de rentrée.md:
```diff
@@ -0,0 +1,84 @@
+
+Les Responsables eProf aident le responsable de Pôle dans la gestion du serveur, la planification et l'organisation des cours.
+
+:small_blue_diamond:**`Composition du Pôle`**
+
+Le Pôle Ef'Réussite s'est restructuré depuis l'année dernière selon ce schéma :
+  - <@890084641317478400>, M2 Network & Cloud Infrastructure, Responsable de Pôle
+  - <@362902765124452364>, L3 classique, Co-Responsable de Pôle
+  - <@542818109912907776>, L3 Int, Responsable eProf L3
+  - <@602744407417946123>, L2 Int, Responsable eProf L2
+  - Un responsable eProf L1, en L1 classique/int/r : MP <@890084641317478400> ou [remplis le form](https://bit.ly/EfRéussiteRecrute) si ça vous intéresse
+  - Un co-responsable eProf L1, en L2 pour pouvoir seconder et accompagner le respo eprof L1 à déterminer les matières qui auraient besoin de cours : MP <@890084641317478400> ou [remplis le form](https://bit.ly/EfRéussiteRecrute) si ça vous intéresse
+  - <@226846557222207488>, M1 Cybersécurité, infrastructure et logiciel, Responsable SharePoint
+
+Tu pourras aussi trouver des membres de l'asso dans le Discord, ils ont le rôle <@&805883190878928907>
+
+
```
Mettre un organigramme (efréussite + respo mentorat + respo vidéo) en image à la place, ça fait plus humain et moins de texte (tu peux laisser en liste les mentions des gens sur l'image comme ça ils sont contactables facilement, mais compresse le en une ligne)
```suggestion
```

gtedavid replied:
je présente vite fait le pôle Ef'Réussite là
s'ils ont des questions ils pourront nous contacter pour en poser 

On pourra présenter le 8 par exemple + en détails Horizon

gtedavid replied:
@lauragossip après que tu as rajouté les photos sur l'organigramme, j'uploaderai l'image et ferai le nécessaire pour l'ajouter à l'annonce

gtedavid replied:
On modifiera légèrement, pour tout avoir sur une ligne avec moins d'infos

iv-stpn commented on resources/global/2022-2023/Annonce de rentrée.md:
```diff
@@ -0,0 +1,84 @@
+On se met aux couleurs de l'association, on sait certains aimer l'ancien logo, on le garde dans notre cœur et on dit bienvenue au nouveau logo !
+
+
+ <@&998640012537122917> <@&998640714281910344>
+Bienvenue à l'Efrei 🥳 
+Sens toi libre de poser tes questions dans les canaux appropriés (<#892407797961089114> pour te repérer)
+Horizon sera présent lors de la journée du 8 septembre, tu pourras poser tes questions ou découvrir les pôles de l'association ;)
+
+
+ <@&794603039477596160> <@&809851024823156736>
+Bienvenue en L2 !
+Allez plus qu'un an avant l'expérience à l'étranger, ça va être une expérience à se raconter des histoires même 10 ans après !
+On charbonne encore
+Le programme de physique a changé l'année dernière et Ef'Réussite a eu du mal à répondre présent 😔, alors c'est **TA chance** pour aider à ce que ça change cette année !
+Dépose des copies, cours, exercices de TD dans les channels docs L1 pour les étudiant(e)s de la Promo 2027, mais aussi pour les années futures ! Tu peux les annonymiser évidemment ;)
+Si tu rencontres des soucis ou n'est pas sûr comment pour accéder ou déposer les documents, contactes <@&934147334760169574>
+
+ <@&794628669737009183> <@&810540179609092096>
+Bienvenue en L... Ah pardon, vous allez à l'étranger 😅 Alors éclatez-vous, mais pas trop, on aimerait vous voir en entier mi-décembre.
+On reste là pour vous épauler 💪. La promo 2023 et 2024 ont quelques annales dans les canaux L3. 
+Vous pouvez demander aussi dans les canaux des destinations pour que les étudiants précédents vous partagent leurs ressources ;)
+
+
+ <@&799636419249045554> <@&810540280155471913>
+Bienvenue en M1, les anciens L3
+Alors ça commence à se professionnaliser ? Trouver un stage c'est difficile 😩 Postuler et postuler disaient-ils...
+Allez on croit en vous, vous avez envie d'évoluer et c'est ça qu'ils cherchent : que vous êtes intéressés par un projet dans l'entreprise, dans l'entreprise même....
+Les connaissances ne font pas tout, le savoir-être aussi ;)
+
+
+Alors oui on a parlé de bosser, mais vous êtes étudiants aussi pour vous éclater et profiter de ces années.
+Avec les autres asso et le BDE, Horizon vous souhaite une bonne rentrée, certes studieuse, mais pleine de nouvelles rencontres et opportunités !
```
A séparer dans les différents channels annonces promos

Ne pas tutoyer comme on ne tutoie pas avant (garde les mêmes textes mais en vouvoyant, je les reverrai plus tard)

gtedavid replied:
vouvoyant corrigé

Pas dans les différents canaux, je pense que c'est mieux qu'on est ouvert à tous quelque soit l'année

iv-stpn commented on resources/global/2022-2023/Annonce de rentrée.md:
```diff
@@ -0,0 +1,84 @@
+
+:small_blue_diamond: **`Pour tout le monde`**
+
+Ci-dessous quelques infos à savoir par rapport à Ef'Réussite et l'Efrei pour bien commencer l'année
+🔸Asso :
+ - les messages dans les canaux <#817741515564122143> et <#892407797961089114> ont été mis à jour
+   Ping <@&701774653940564078> si quelque chose n'est pas clair
+ - les rôles pour accéder au serveur restent les mêmes : Efrei → Pacours Ingé/Pex → Rôles Ingé/Pex, ainsi qu'un canal de rôles supplémentaires
+ - On n'est jamais fermé ! On est ouvert à toutes et à tous, curieux, soucieux d'aider sans savoir comment. On finit toujours par trouver un nouveau moyen innovant
+ - Ah oui on a un bot et .... *(whispers :shushing_face:)* ah ! On ne peut pas vous en dire plus 🤷‍♂️. Restez aux aguets ;)
+🔸Efrei : 
+- le **responsable réussite étudiante de l'EFREI** devient le **Responsable de Formation**
+
+Et qui dit rentrée.... Dit nouveautés !
+On se met aux couleurs de l'association, on sait certains aimer l'ancien logo, on le garde dans notre cœur et on dit bienvenue au nouveau logo !
```
Créer un channel "changelog" pour le serveur (c'est pertinent que pour les intéressés, et c'est bien de tout archiver au même endroit)

Mettre en valeur le nouveau logo (gras souligné)

gtedavid replied:
@lauragossip @noftaly @Kadiatou-yatt vous en pensez quoi ?

SaadaLaura replied:
Maybe si on a fait l'émoji du nouveau logo on peut le mettre à côté de "nouveautés" 
Apres est-ce que c'est vraiment utile un "changelog" perso je pense pas (je suis plutôt neutre)

noftaly replied:
Perso je ne pense pas que ce qui soit dit ici soit vraiment nécessaire, tout peut être enlevé... après si vous voulez que ce soit dit qqpart peut etre le raccourcir (tous les points dans "Asso :" sont facultatif et peu important) pour le mettre a la fin de ce message, car c'est vrai qu'un salon Changelog c'est peut etre overkill

iv-stpn reviewed (changes requested):
(approuvé par erreur: prendre en compte les changements précédents) :)

gtedavid committed:
! into smile

Co-authored-by: Ivan STEPANIAN <iv.stpn@gmail.com>

gtedavid committed:
long to short

Co-authored-by: Ivan STEPANIAN <iv.stpn@gmail.com>

gtedavid committed:
shorter

Co-authored-by: Ivan STEPANIAN <iv.stpn@gmail.com>

gtedavid committed:
supp pr étudiants L7

Co-authored-by: Ivan STEPANIAN <iv.stpn@gmail.com>

noftaly reviewed (changes requested):
Je suis accord avec les suggestions d'Ivan donc si elles sont bonnes c'est good pour moi 👍🏼
Il reste donc notamment a séparer les messages spécifiques aux promo dans des fichiers à part

PS: regarder [ici pour intégrer une image](https://github.com/horizon-efrei/efreussite-webhooks#-astuces-pour-un-message-bien-formatté) (prendre exemple sur celles déjà ajoutées)

Et se souvenir qu'un message ne peux pas faire plus de 2000 caractères (emojis, liens, mentions, formattage (`**`...) compris). Utiliser `===MESSAGE_BREAK===` pour séparer les messages. Penser au faite que ca peut double-mentionner des personnes si un de ses rôles est mentionné dans deux messages différents

gtedavid committed:
V2 Annonce

Reformulation et organisation des différentes parties

gtedavid commented:
J'ai remanié plusieurs parties
dites moi ce que vous en pensez

Je sais l'image n'est pas correcte, je corrige dès que je l'ai

noftaly commented on resources/global/2022-2023/Annonce de rentrée.md:
```diff
@@ -0,0 +1,81 @@
+
+IMAGE
+
+Les canaux ` 🟠-cours-LX` comportent les annonces des cours auxquels tu peux t'inscrire aux rappels 🔔
```
```suggestion
Les canaux `🟠-cours-LX` comportent les annonces des cours auxquels tu peux t'inscrire 🔔
```

noftaly commented on resources/global/2022-2023/Annonce de rentrée.md:
```diff
@@ -0,0 +1,81 @@
+
+On espère que vous avez passé un bel été et nous vous souhaitons une bonne rentrée !
+
+:small_blue_diamond: **`Pour les nouveaux 🌱 : Précisions sur l'organisation du Pôle`**
```
```suggestion
:small_blue_diamond: **`Pour les nouveaux 🌱 : précisions sur l'organisation du Pôle`**
```

gtedavid replied:
Ah aussi, je pensais mettre pour les nouveaux ou les un peu perdus
mais ça fait pas trop le "un peu" perdus.... @noftaly

gtedavid replied:
```suggestion
:small_blue_diamond: **`Pour les nouveaux 🌱 ou ceux qui nous connaissent pas : précisions sur l'organisation du Pôle`**
```

gtedavid replied:
```suggestion
:small_blue_diamond: **`Pour les nouveaux 🌱 ou ceux qui sont dans le flou : précisions sur l'organisation du Pôle`**
```

noftaly commented on resources/global/2022-2023/Annonce de rentrée.md:
```diff
@@ -0,0 +1,81 @@
+:small_blue_diamond: **`Pour les nouveaux 🌱 : Précisions sur l'organisation du Pôle`**
+
+Ef'Réussite est le serveur d'entraide étudiante de l'Efrei.
+Pôle de l'association <@&805883190878928907>, il permet ainsi à tout étudiant voulant donner de son temps pour aider ses camarades ou d'autres étudiants qui auraient des questions ou des difficultés, en échange d'une reconnaissance pour sa participation dans cet objectif d'entraide... 
```
Je trouve que c'est très long pour pas grand chose, en somme on veut juste dire que "Ef'Réussite, géré par Horizon, permet à des étudiants de s'entraider."
Je trouve que la formulation d'Ivan était bonne
> Il est géré par l'association <@&805883190878928907>, par des étudiants bénévoles souhaitant aider leurs pairs (appelés <@&805873100813500447> ou eProfs).

C'est plus succinct ; plus facile à lire et c'est important de garder cette annonce courte et compréhensible en lecture rapide 😄 

```suggestion
Il est géré par l'association <@&805883190878928907>, par des étudiants bénévoles souhaitant aider leurs pairs (appelés <@&805873100813500447> ou eProfs).
```

gtedavid replied:
J'aime pas trop la formulation, donc je mets de nouveau en statut "en réflexion"

noftaly commented on resources/global/2022-2023/Annonce de rentrée.md:
```diff
@@ -0,0 +1,81 @@
+Ef'Réussite est le serveur d'entraide étudiante de l'Efrei.
+Pôle de l'association <@&805883190878928907>, il permet ainsi à tout étudiant voulant donner de son temps pour aider ses camarades ou d'autres étudiants qui auraient des questions ou des difficultés, en échange d'une reconnaissance pour sa participation dans cet objectif d'entraide... 
+
+Les <@&934147334760169574> gèrent Ef'Réussite avec les <@&938679049746792448> ; contactez-les pour toute question relative à la planification et l'organisation de cours, ou pour vous [engager](https://bit.ly/EfRéussiteRecrute) comme <@&805873100813500447> !
```
```suggestion
Les <@&934147334760169574> gèrent Ef'Réussite avec les <@&938679049746792448> ; contactez-les pour toute question relative à la planification et l'organisation de cours, ou [pour vous engager comme](https://bit.ly/EfRéussiteRecrute) <@&805873100813500447> !
```
j'ai juste agrandi le lien pour qu'il soit + visible et + facile a cliquer 😄

gtedavid replied:
> j'ai juste agrandi le lien pour qu'il soit + visible et + facile a cliquer 😄

j'aurais dit même l'étendre au rôle le lien....

J'aime pas trop la formulation, donc je mets de nouveau en statut "en réflexion"

gtedavid replied:
mb trompé de suggestions 😭

gtedavid replied:
> j'ai juste agrandi le lien pour qu'il soit + visible et + facile a cliquer 😄

j'aurais dit même l'étendre au rôle le lien....
Tu en penses quoi @noftaly ?

gtedavid replied:
```suggestion
Les <@&934147334760169574> gèrent Ef'Réussite avec les <@&938679049746792448> ; contactez-les pour toute question relative à la planification et l'organisation de cours, ou [pour vous engager comme <@&805873100813500447>](https://bit.ly/EfRéussiteRecrute) !
```

noftaly commented on resources/global/2022-2023/Annonce de rentrée.md:
```diff
@@ -0,0 +1,81 @@
+IMAGE
+
+Les canaux ` 🟠-cours-LX` comportent les annonces des cours auxquels tu peux t'inscrire aux rappels 🔔
+Les cannaux des ressources pédagogiques sont identifiables par la présence "-docs" 📄
```
```suggestion
Les canaux des ressources pédagogiques comportent le suffixe "-docs" 📄
```

noftaly commented on resources/global/2022-2023/Annonce de rentrée.md:
```diff
@@ -0,0 +1,81 @@
+Les cours ont lieu sur Discord ou Teams et sont généralement enregistrés par les <@&805873100813500447>. Ils sont ensuite mis sur [SharePoint d'Horizon](https://bit.ly/EfRéussiteSP) * 
+Si vous souhaitez qu'une de vos interventions soit supprimée du rendu public de la vidéo, que votre nom n'y apparaisse pas, ou autre, envoyez une demande au <@&1012394085174227045>.
+
+**NOTE: des enseignants et membres de l'administration de l'Efrei sont présents sur le Discord.**
```
```suggestion
**NOTE : des enseignants et membres de l'administration de l'Efrei sont présents sur le Discord.**
```

noftaly commented on resources/global/2022-2023/Annonce de rentrée.md:
```diff
@@ -0,0 +1,81 @@
+Les canaux ` 🟠-cours-LX` comportent les annonces des cours auxquels tu peux t'inscrire aux rappels 🔔
+Les cannaux des ressources pédagogiques sont identifiables par la présence "-docs" 📄
+
+Les cours ont lieu sur Discord ou Teams et sont généralement enregistrés par les <@&805873100813500447>. Ils sont ensuite mis sur [SharePoint d'Horizon](https://bit.ly/EfRéussiteSP) * 
```
```suggestion
Les cours ont lieu sur Discord ou Teams et sont généralement enregistrés par les <@&805873100813500447>. Ils sont ensuite mis sur [SharePoint d'Horizon](https://bit.ly/EfRéussiteSP)¹.
```

noftaly commented on resources/global/2022-2023/Annonce de rentrée.md:
```diff
@@ -0,0 +1,81 @@
+
+
+🔹 **`On recrute !`**
+Comme l'année précédente, on recherche un responsable eProf L1, en L1 classique/int/r, ainsi qu'un co-responsable eProf L1, essentiellement en L2 pour pouvoir seconder et accompagner le respo eprof L1 à déterminer les matières qui auraient besoin de cours.
```
```suggestion
Comme l'année précédente, on recherche un <@&934144211815661658>, en L1 classique/int/r, ainsi qu'un co-responsable eProf L1, en L2 pour pouvoir accompagner le <@&934144211815661658> et l'aider à déterminer les matières qui auraient besoin de cours.
```

noftaly commented on resources/global/2022-2023/Annonce de rentrée.md:
```diff
@@ -0,0 +1,81 @@
+🔹 **`On recrute !`**
+Comme l'année précédente, on recherche un responsable eProf L1, en L1 classique/int/r, ainsi qu'un co-responsable eProf L1, essentiellement en L2 pour pouvoir seconder et accompagner le respo eprof L1 à déterminer les matières qui auraient besoin de cours.
+
+On souhaite pouvoir développer et organiser l'entraide aux étudiants du Parcours Experts du Numérique. Si vous êtes intéressés, contactez-nous ou remplissez le [form](https://bit.ly/EfRéussiteRecrute) !
```
```suggestion
On souhaite pouvoir développer et organiser l'entraide aux étudiants <@&809188399248179201>. Si vous êtes intéressés, contactez-nous ou [remplissez le formulaire](https://bit.ly/EfRéussiteRecrute) !
```

noftaly commented on resources/global/2022-2023/Annonce de rentrée.md:
```diff
@@ -0,0 +1,81 @@
+
+Ci-dessous quelques infos à savoir par rapport à Ef'Réussite et l'Efrei pour bien commencer l'année
+🔸Asso :
+ - les messages dans les canaux <#817741515564122143> et <#892407797961089114> ont été mis à jour
+   Ping <@&701774653940564078> si quelque chose n'est pas clair
+ - Ef'Réussite expérimente depuis juin la réalisation de cours en présentiel (5, 10 à 20 étudiants) pour favoriser l'entraide entre __étudiants__. 
+ - les rôles pour accéder au serveur restent les mêmes : Efrei → Pacours Ingé/Pex → Rôles Ingé/Pex, ainsi qu'un canal de rôles supplémentaires
+ - On n'est jamais fermé ! On est ouvert à toutes et à tous, curieux, soucieux d'aider sans savoir comment. On finit toujours par trouver un nouveau moyen innovant
+ - Ah oui on a un bot et .... *(whispers :shushing_face:)* ah ! On ne peut pas vous en dire plus 🤷‍♂️. Restez aux aguets ;)
+ - Une annonce sur les Pôles d'Horizon sera faite prochainement 📢
```
Je trouve qu'aucune de ces annonces n'est pertinente à part la 2ème... Essentiellement la première n'est pas utile, les gens ne re-liront pas de toute façon... La deuxième est pertinente, la 3ème ne l'est pas, dire que 3 rôles n'ont pas changé n'est pas utile 😄 la 4ème non plus, la 5ème pourquoi pas, à voir s'il y a besoin de la garder, et sinon juste attendre la vraie annonce. La 6ème aussi, ce n'est pas utile de dire "on va bientôt faire une annonce" 😄 

Et de manière générale je ne trouve pas que ces infos "aident a bien commencer l'année" 😅 

```suggestion
 - Ef'Réussite expérimente depuis juin la réalisation de cours en présentiel (5, 10 à 20 étudiants) pour favoriser l'entraide entre __étudiants__. 
```

S'il ne reste plus que ca, il faudrait peut être revoit la section pour la rédiger différemment du coup (sans liste) ou l'intégrer a des parties avant

gtedavid replied:
ok je prends compte de tes suggestions 😅 
j'applique et je reprendrai si j'ai le temps

gtedavid replied:
Corrigé suite à une discussion avec Laura

gtedavid replied:
Corrigé suite à une discussion avec Laura 2

noftaly commented on resources/global/2022-2023/Annonce de rentrée.md:
```diff
@@ -0,0 +1,81 @@
+Allez bonne rentrée à toutes et à tous 🥳👌
+
+
+* `'*' si vous avez des problèmes d'affichage des vidéos, vous pouvez demander qu'on vous envoie un lien sur votre adresse efrei.net. Le problème est connu. L'équipe a contacté le service informatique de l'Efrei pour trouver une solution viable...`
```
```suggestion
* `¹: si vous avez des problèmes d'affichage des vidéos, vous pouvez demander qu'on vous envoie un lien sur votre adresse efrei.net. Le problème est connu. L'équipe a contacté le service informatique de l'Efrei pour trouver une solution viable...`
```

gtedavid replied:
faut que je le rajoute à côté du * 
jsp comment faire les 1 en exposant....

noftaly commented on resources/global/2022-2023/Annonce de rentrée.md:
```diff
@@ -0,0 +1,81 @@
+On se met aux couleurs de l'association, on sait certains aimer l'ancien logo, on le garde dans notre cœur et on dit bienvenue au nouveau logo !
+
+
+ <@&998640012537122917> <@&998640714281910344>
+Bienvenue à l'Efrei 🥳 
+Sentez-vous libre de poser vos questions dans les canaux appropriés (<#892407797961089114> pour vous repérer)
+Horizon sera présent lors de la journée du 8 septembre, vous pourrez poser vos questions ou découvrir les pôles de l'association ;)
+
+
+ <@&794603039477596160> <@&809851024823156736>
+Bienvenue en L2 !
+Allez plus qu'un an avant l'expérience à l'étranger, ça va être une expérience à se raconter des histoires même 10 ans après !
+On charbonne encore
+Le programme de physique a changé l'année dernière et Ef'Réussite a eu du mal à répondre présent 😔, alors c'est **VOTRE chance** pour aider à ce que ça change cette année !
+Déposez des copies, cours, exercices de TD dans les channels docs L1 pour les étudiant(e)s de la Promo 2027, mais aussi pour les années futures ! Vous pouvez les annonymiser évidemment ;)
+Si vous rencontrez des soucis ou n'est pas sûr comment pour accéder ou déposer les documents, contactes <@&934147334760169574>
+
+ <@&794628669737009183> <@&810540179609092096>
+Bienvenue en L... Ah pardon, vous allez à l'étranger 😅 Alors éclatez-vous, mais pas trop, on aimerait vous voir en entier mi-décembre.
+On reste là pour vous épauler 💪. La promo 2023 et 2024 ont quelques annales dans les canaux L3. 
+Vous pouvez demander aussi dans les canaux des destinations pour que les étudiants précédents vous partagent leurs ressources ;)
+
+
+ <@&799636419249045554> <@&810540280155471913>
+Bienvenue en M1, les anciens L3
+Alors ça commence à se professionnaliser ? Trouver un stage c'est difficile 😩 Postuler et postuler disaient-ils...
+Allez on croit en vous, vous avez envie d'évoluer et c'est ça qu'ils cherchent : que vous êtes intéressés par un projet dans l'entreprise, dans l'entreprise même....
+Les connaissances ne font pas tout, le savoir-être aussi ;)
+
+
+Alors oui on a parlé de bosser, mais vous êtes étudiants aussi pour vous éclater et profiter de ces années.
+Avec les autres asso et le BDE, Horizon vous souhaite une bonne rentrée, certes studieuse, mais pleine de nouvelles rencontres et opportunités !
+
+Allez bonne rentrée à toutes et à tous 🥳👌
+
```
Mettre dans des annonces à part pour les promos concernées (tu peux tout faire dans cette PR, tu as juste a créer les fichiers etc sur ta branche et ca va automatiquement tout ajouter à cette PR)

gtedavid replied:
je vais mettre des ==msg break==
je pense que ça suffira... on verra

noftaly commented on resources/global/2022-2023/Annonce de rentrée.md:
```diff
@@ -0,0 +1,81 @@
+- le **responsable réussite étudiante de l'EFREI** devient le **Responsable de Formation**
+
+Et qui dit rentrée.... Dit nouveautés !
+On se met aux couleurs de l'association, on sait certains aimer l'ancien logo, on le garde dans notre cœur et on dit bienvenue au nouveau logo !
```
```suggestion
Nous nous mettons aux couleurs de l'association, nous savons que certains aimaient l'ancien logo du serveur ; on le garde dans notre cœur et on dit bienvenue au nouveau logo !
```

melisandeGB commented on resources/global/2022-2023/Annonce de rentrée.md:
```diff
@@ -0,0 +1,81 @@
+Allez plus qu'un an avant l'expérience à l'étranger, ça va être une expérience à se raconter des histoires même 10 ans après !
+On charbonne encore
+Le programme de physique a changé l'année dernière et Ef'Réussite a eu du mal à répondre présent 😔, alors c'est **VOTRE chance** pour aider à ce que ça change cette année !
+Déposez des copies, cours, exercices de TD dans les channels docs L1 pour les étudiant(e)s de la Promo 2027, mais aussi pour les années futures ! Vous pouvez les annonymiser évidemment ;)
```
```suggestion
Déposez des copies, cours, exercices de TD dans les channels docs L1 pour les étudiant(e)s de la Promo 2027, mais aussi pour les années futures ! Vous pouvez les anonymiser évidemment ;)
```

melisandeGB commented on resources/global/2022-2023/Annonce de rentrée.md:
```diff
@@ -0,0 +1,81 @@
+On charbonne encore
+Le programme de physique a changé l'année dernière et Ef'Réussite a eu du mal à répondre présent 😔, alors c'est **VOTRE chance** pour aider à ce que ça change cette année !
+Déposez des copies, cours, exercices de TD dans les channels docs L1 pour les étudiant(e)s de la Promo 2027, mais aussi pour les années futures ! Vous pouvez les annonymiser évidemment ;)
+Si vous rencontrez des soucis ou n'est pas sûr comment pour accéder ou déposer les documents, contactes <@&934147334760169574>
```
```suggestion
Si vous rencontrez des soucis ou ne savez pas comment accéder aux documents ou où les déposer, contactez <@&934147334760169574>
```

melisandeGB commented on resources/global/2022-2023/Annonce de rentrée.md:
```diff
@@ -0,0 +1,81 @@
+
+ <@&794628669737009183> <@&810540179609092096>
+Bienvenue en L... Ah pardon, vous allez à l'étranger 😅 Alors éclatez-vous, mais pas trop, on aimerait vous voir en entier mi-décembre.
+On reste là pour vous épauler 💪. La promo 2023 et 2024 ont quelques annales dans les canaux L3. 
```
```suggestion
On reste là pour vous épauler 💪. Les promos 2023 et 2024 ont quelques annales dans les canaux L3. 
```

melisandeGB commented on resources/global/2022-2023/Annonce de rentrée.md:
```diff
@@ -0,0 +1,81 @@
+
+
+Alors oui on a parlé de bosser, mais vous êtes étudiants aussi pour vous éclater et profiter de ces années.
+Avec les autres asso et le BDE, Horizon vous souhaite une bonne rentrée, certes studieuse, mais pleine de nouvelles rencontres et opportunités !
```
```suggestion
Avec les autres assos et le BDE, Horizon vous souhaite une bonne rentrée, certes studieuse, mais pleine de nouvelles rencontres et opportunités !
```

gtedavid commented:
> Je suis accord avec les suggestions d'Ivan donc si elles sont bonnes c'est good pour moi 👍🏼 Il reste donc notamment a séparer les messages spécifiques aux promo dans des fichiers à part
> 
> PS: regarder [ici pour intégrer une image](https://github.com/horizon-efrei/efreussite-webhooks#-astuces-pour-un-message-bien-formatt%C3%A9) (prendre exemple sur celles déjà ajoutées)
> 
> Et se souvenir qu'un message ne peux pas faire plus de 2000 caractères (emojis, liens, mentions, formattage (`**`...) compris). Utiliser `===MESSAGE_BREAK===` pour séparer les messages. Penser au faite que ca peut double-mentionner des personnes si un de ses rôles est mentionné dans deux messages différents

j'avais oublié que les 2 sauts de ligne ne fonctionnaient plus 😅 
merci pour le rappel, je modifie ça après avoir lu vos suggestions

gtedavid committed:
Enlever la notion de rappel L cours LX

Co-authored-by: Elliot <elliot.maisl@gmail.com>

gtedavid committed:
Pas de majuscules Précisions pour les nouvaux

Co-authored-by: Elliot <elliot.maisl@gmail.com>

gtedavid committed:
Canaux Docs - suffixes

Co-authored-by: Elliot <elliot.maisl@gmail.com>

gtedavid committed:
Ajustements sur le msg d'ouverture aux Pex

Co-authored-by: Elliot <elliot.maisl@gmail.com>

gtedavid committed:
New stuff reduction

Co-authored-by: Elliot <elliot.maisl@gmail.com>

gtedavid committed:
Remise de l'info sur les rôles

gtedavid committed:
Update news suite à discussion avec Laura 2

gtedavid committed:
vouvoiement et corrections des si vous avez des soucis

Co-authored-by: Mélisande  <72623143+melisandeGB@users.noreply.github.com>

gtedavid committed:
on -> nous au niv des couleurs asso

Co-authored-by: Elliot <elliot.maisl@gmail.com>

gtedavid committed:
anonymiser correction écriture

Co-authored-by: Mélisande  <72623143+melisandeGB@users.noreply.github.com>

gtedavid committed:
pluriel pour les promo 2023 et 2024

Co-authored-by: Mélisande  <72623143+melisandeGB@users.noreply.github.com>

gtedavid committed:
pluriel assos et BDE

Co-authored-by: Mélisande  <72623143+melisandeGB@users.noreply.github.com>

gtedavid committed:
exposant 1 en bas d'annonce

Co-authored-by: Elliot <elliot.maisl@gmail.com>

gtedavid committed:
exposant 1 sur SP

gtedavid committed:
Rallongement du lien cliquable

gtedavid committed:
Update Annonce de rentrée.md

gtedavid committed:
Message break added

gtedavid committed:
Adding Organigramme

gtedavid committed:
Corrections finales
